tests/gadget-zero/Jenkinsfile

```diff
@@ -64,7 +64,7 @@ pipeline {
         		. .env3/bin/activate
         		cd tests/gadget-zero
         		rm -rf tests
-        		python test_gadget0.py -X
+        		python3 test_gadget0.py -X
```

Comment:
this actually runs in a venv, see line 26 and 64, so this is "fine" but unnecessary